packages/jsActions/native-mobile-resources/config/rollup.config.js

```diff
@@ -0,0 +1,114 @@
+import { red, yellow } from "colors";
+import { mkdirSync, promises } from "fs";
+import { join, relative } from "path";
+import clear from "rollup-plugin-clear";
+import command from "rollup-plugin-command";
+import { cp } from "shelljs";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import { collectNativeDependencies } from "../../../tools/pluggable-widgets-tools/configs/rollup-plugin-collect-native-dependencies";
+
+const cwd = process.cwd();
+
+export default async args => {
+    const jsActionTargetFolder = `javascriptsource/${args.configProject ?? "nativemobileresources"}/actions`;
+    const result = [];
+    const files = await findFiles();
+    const outDir = join(cwd, "dist");
+
+    files.forEach((file, i) => {
+        const fileInput = file.path.replace(join(cwd, "/"), "");
+        const [fileOutput] = file.name.split(".");
+        result.push({
+            input: fileInput,
+            output: {
+                format: "es",
+                file: join(outDir, `${fileOutput}.js`),
+                sourcemap: false
+            },
+            external: nativeExternal,
+            plugins: [
+                ...(i === 0 ? [clear({ targets: [outDir] })] : []),
+                collectNativeDependencies(nativeExternal, outDir, true, fileOutput, i === 0),
+                nodeResolve({ preferBuiltins: false, mainFields: ["module", "browser", "main"] }),
+                typescript({
+                    noEmitOnError: false,
+                    sourceMap: false,
+                    inlineSources: false,
+                    target: "es2019",
+                    types: ["mendix-client", "big.js", "react-native"],
+                    allowSyntheticDefaultImports: true
+                }),
+                command([
+                    () => {
+                        if (i === files.length - 1) {
+                            const destinationFolder = join(cwd, "./tests/testProject/", jsActionTargetFolder);
```

Comment:
There's no need to include a `./` prefix or a trailing `/` for a path element in `join`, right?

Reply:
Fixed

Reply:
*PS: File for jsaction now is on root/configs/jsactions